styles/sass/_scratch.scss

```diff
@@ -0,0 +1,255 @@
+// Main Blue: #0b69ff
+// Navy Blue: #171f46
+// Body Grey: #7e858e
+// Border Grey: #d7dfe9
+
+html {
+    font-size: 62.5%;
+    font-family: 'Inter', Arial, 'Helvetica Neue', Helvetica, sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+    margin: 0;
+}
+
+h2,
+h3 {
+    font-weight: 500;
+    font-size: 2.4rem;
+    letter-spacing: 0.02rem;
+    line-height: 1.25;
+}
+
+p {
+    margin: 0;
+    // border: 1px solid red;
+}
+
+a {
+    text-decoration: none;
+}
+
+ul {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+img {
+    max-width: 100%;
+    display: block;
+}
+
+fieldset {
+    border: none;
+    padding: 0;
+}
+
+.wrapper {
+    max-width: 1680px;
+    width: 90%;
+    margin: 0 auto;
+}
+
+.button {
+    background-color: #0b69ff;
+    cursor: pointer;
+    color: #fff;
+    width: 340px;
+    height: 60px;
+    font-family: 'Inter', Arial, 'Helvetica Neue', Helvetica, sans-serif;
+    font-size: 2rem;
+    font-weight: 600;
+    letter-spacing: 0.05rem;
+    line-height: 1.4;
+    border: none;
+    border-radius: 6px;
+    -webkit-border-radius: 6px;
+    -moz-border-radius: 6px;
+    -ms-border-radius: 6px;
+    -o-border-radius: 6px;
+}
+
+.button:hover {
+    background: #0058E3;
```

Comment:
Remember we have the power of [Sass variables](https://sass-lang.com/documentation/variables). We should stick to using those for colors, fonts & reused things like border radius/box-shadow.